src/query_executor.c

```diff
@@ -83,6 +98,27 @@ static void _replicateMergeClauseToMatchClause(AST *ast) {
     ast->matchNode = New_AST_MatchNode(wrappedEntities);
 }
 
+/* Create a RETURN clause from CALL clause. */
+static void _replicateCallClauseToReturnClause(AST *ast) {
+    assert(ast->callNode && !ast->returnNode);
+
+    AST_ProcedureCallNode *call = ast->callNode;
+    unsigned int output_len = array_len(call->yield);
+    AST_ReturnElementNode **returnElements = array_new(AST_ReturnElementNode*, output_len);
+
+    for(unsigned int i = 0; i < output_len; i++) {
+        char *alias = NULL;
+        char *property = NULL;
+        char *output = call->yield[i];
+        AST_ArithmeticExpressionNode *exp = New_AST_AR_EXP_VariableOperandNode(output, property);
+        AST_ReturnElementNode *returnElement = New_AST_ReturnElementNode(exp, alias);
+        returnElements = array_append(returnElements, returnElement);
+    }
+
+    int distinct = 0;
```

Comment:
variable seems unnecessary

Reply:
This is just so `New_AST_ReturnNode(returnElements, distinct);` will be clearer.